include/libtorrent/alert_types.hpp

```diff
@@ -2064,6 +2065,12 @@ namespace libtorrent
 		boost::array<char, 64> signature;
 		std::string salt;
 		boost::uint64_t seq;
+
+		// DHT put operation usually writes item to k nodes, maybe the node
+		// is stale so no response, or the node doesn't support 'put', or the
+		// token for write is out of date, etc.
+		// set success to true when success responses >= failed responses.
```

Comment:
does no response at all count as a "failed response"? This condition seems a bit arbitrary. wouldn't it make more sense to just include the number of confirmed successful responses instead?